.settings/JMRI PanelPro.launch

```diff
@@ -16,6 +16,7 @@
 <booleanAttribute key="M2_WORKSPACE_RESOLUTION" value="false"/>
 <booleanAttribute key="org.eclipse.debug.ui.ATTR_LAUNCH_IN_BACKGROUND" value="false"/>
 <listAttribute key="org.eclipse.debug.ui.favoriteGroups">
+<listEntry value="org.eclipse.debug.ui.launchGroup.debug"/>
```

Comment:
Was this an intentional addition?  (I don't know enough about Eclipse to know whether it maters to have this in the `launchGroup.debug` or not)